lib/matplotlib/axes/_axes.py

```diff
@@ -188,7 +188,8 @@ def get_xlabel(self):
         label = self.xaxis.get_label()
         return label.get_text()
 
-    def set_xlabel(self, xlabel, fontdict=None, labelpad=None, **kwargs):
+    def set_xlabel(self, xlabel, fontdict=None, labelpad=None, loc=None,
```

Comment:
Let's keep this keyword-only to reduce the API-Footprint:
```suggestion
    def set_xlabel(self, xlabel, fontdict=None, labelpad=None, *, loc=None,
```